code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -780,7 +892,7 @@ construct_runtime!(
 		Tokens: orml_tokens = 52,
 		CurrencyFactory: currency_factory = 53,
 		GovernanceRegistry: governance_registry = 54,
-		Assets: assets = 55,
+		// Assets: assets = 55,
```

Comment:
In general, it's better to just remove code than to comment it out. If there's a good reason to keep it, it should probably have a comment justifying why it's still there (TODO, etc)

Reply:
I removed this.